### ISSUE TITLE
A new feature of adding entertainment resources 

### ISSUE THREAD

**Issue by shrutibharat-05** [Mansi168/EmoWell#221]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Hey in Emowell application we can provide users a source of entertainment as per their preferences.
Also,we can categorize users according to their age in order to provide them best help needed. 

### Use Case

We can broaden the horizon of users using this app to all age group people.
And, adding extra feature(entertainment resource)  mentioned above would give more user engagement.

### Priority

High

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [ ] I'm willing to provide further clarification or assistance if needed.

**Comment by everly-gif** [comment]
Hi, Great idea ! But Please be more specific, what kind of entertainment resource will you add ? We already have games as a source of entertainment

**Comment by shrutibharat-05** [comment]
Yeah, I meant games would only be restricted to specific age-group only. So for other people accessing this application should get some other resources that align with their area of interest.

**Comment by everly-gif** [comment]
Alright? Like what resources do you plan to add, give examples

**Comment by shrutibharat-05** [comment]
We can categorize the entertainment resources. 
Besides games, 1)Old movies, 2)Songs 3)Karyoke singing option(provide related website name) 4)Interesting E-books  5)Creative space where people can write or create any creative stuff and share them under Creative section.

Something like this ,if works.

**Comment by Mansi168** [comment]
ok, you can start working on it, but please make sure your branch does not have conflicts.
